chore: Minor improvements to CONTRIBUTING.md (#3447)
mongodb/mongodb-atlas-cli: CONTRIBUTING.md

#### Environment

- Fork the repository.
- Clone your fork locally.

> - Set up development environment:

Reviewer: ```suggestion
- Set up your development environment:
```

Author: Added 👍

  - We use Go Modules to manage dependencies, so you can develop outside your `$GOPATH`.

>   - Run `make setup` to install required dependencies and developer tools. This includes [golangci-lint](https://github.com/golangci/golangci-lint), which we use to lint our code.

Reviewer: ```suggestion
  - Run `make setup` to install required dependencies and developer tools including [golangci-lint](https://github.com/golangci/golangci-lint), which we use to lint our code.
```

Author: Added

>   - Run `make setup` to install required dependencies and developer tools. This includes [golangci-lint](https://github.com/golangci/golangci-lint), which we use to lint our code.

Reviewer 2: should we also mention any of those two commands:
```
deps                           Download go module dependencies
devtools                       Install dev tools
```

Author: Looking into the makefile, `make setup` already runs deps, devtools and link-git-hooks so I don't think it is needed.

Reviewer 2: ah, perfect! thanks

    "go.testTags": "unit,e2e"
```

This will enable compilation for unit and end-to-end tests.

#### Debugging in VSCode

> To debug in VSCode, you need to create a debug configuration for the command with required arguments.
> Run the following commands to create a new launch.json file for the debugger:

Reviewer: ```suggestion
To debug in VSCode, you must create a debug configuration for the command with the required arguments.
Run the following commands to create a new launch.json file for the debugger:
```

Author: Added

```
touch .vscode/launch.json
```

> Then put the following example configuration into the file.
> Review and replace the command name and arguments depending on the command you wish to debug.

Reviewer: ```suggestion
Then put the following example configuration into the file.
Review and replace the command name and arguments depending on the command you want to debug.
```

Author: Added

```json
{
    "configurations": [
        {
            "name": "DBuser Create Command",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${workspaceFolder}/cmd/atlas",
            "env": {},
            "args": [
              "dbuser",
              "create",
              "atlasAdmin", 
              "--username",
              "myUser",
              "--password",

>               "123abc"

Author: I suggest changing the example configuration to something with more args and flags required to more clearly show how the configuration needs to be altered for alternate commands.
